pkg/ops/iso.go

```diff
@@ -166,6 +166,11 @@ func GenISO(srcFunc, dstFunc valueGetOnCall, i schema.ISO) func(ctx context.Cont
 		if err != nil {
 			internal.Log.Logger.Error().Msgf("Failed generating iso '%s' from '%s'. Error: %s", i.Name, src, err.Error())
 		}
+
+		if err := cleanupTempFiles(dst); err != nil {
```

Comment:
there is a step `CleanTmpDirs` that should clean everything, maybe we just have to add it to the deployer and call it a day?